frontend/src/components/Post.jsx

```
@@ -458,7 +430,7 @@ return (
           onClick={() => handleLike(post.id)}
           className="flex items-center text-gray-700"
         >
-          {liked ? <FaHeart className="text-red-500" /> : <FaRegHeart className="text-gray-500" />}
+          {liked ? <FaThumbsUp className="text-blue-600" /> : <FaThumbsUp className="text-gray-600" />}
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)